package.json

```diff
@@ -28,5 +31,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
Love that you put this in `devDependencies`! A more naive approach would have just to have made this a regular dependency, but that would have added quite a lot of bloat to the compiled JavaScript. Good job!